tests/test_sofa.py

```diff
@@ -91,25 +91,25 @@ def test_list_dimensions(capfd):
 
     # test FIR Data
     sofa = sf.Sofa("GeneralFIR")
-    sofa.list_dimensions    # noqa: B018
+    _ = sofa.list_dimensions
```

Comment:
Haha, why did I not think of this? Thx.

Reply:
I came across it in the documentation; I hadn’t thought of it either! 😂